src/Backend/Modules/Blog/Actions/Add.php

```diff
@@ -125,15 +125,6 @@ private function validateForm()
                 $this->frm->getField('category_id')->addError(BL::err('FieldIsRequired'));
             }
 
-            if ($this->imageIsAllowed) {
```

Comment:
ah, you are right indeed, I saw the isFilled but overlooked that it didn't throw an error when empty